### ISSUE TITLE
Bad interwiki prefixes added while editing dashboard related pages by Dashboard

### ISSUE THREAD

**Issue by urbanecm** [WikiEducationFoundation/WikiEduDashboard#2128]
Hi, 

I was informed Dashboard is using bad interwiki prefixes (always directing to English language project regardless on project language. 

This is example: https://cs.wikipedia.org/w/index.php?title=Wikipedie:N%C3%A1st%C4%9Bnka/Wikimedia_%C4%8CR/Editaton_v_%C4%8Cesk%C3%BDch_Bud%C4%9Bjovic%C3%ADch_(podzim_2018)&diff=16547465&oldid=16546868 . 

Fix: https://cs.wikipedia.org/w/index.php?title=Wikipedie:N%C3%A1st%C4%9Bnka/Wikimedia_%C4%8CR/Editaton_v_%C4%8Cesk%C3%BDch_Bud%C4%9Bjovic%C3%ADch_(podzim_2018)&diff=16547473&oldid=16547465 (I know fixing things after Dashboard is useless, this is just for demonstration). 

In short, instead of `[[:language:family:page]]` Dashboard should use `[[:family:language:page]]`. This is fully qualified link and will work in all WMF projects. 

Thank you for fixing this, 

Martin Urbanec
cs.wikipedia admin

**Comment by JAnDud** [comment]
and family should be shortened to standard interwiki prefix:
wikisource:->s:
wiktionary:->wikt:
wikiquote:->q:
etc.

**Comment by urbanecm** [comment]
That's another possible solution, but wikisource: and s: is the same (on WMF wikis), and as nobody should edit dashboard-maintained page manually and insert the prefix manually, it can do no harm I think.

**Comment by ragesoss** [comment]
Just deployed a fix, and it appears to be working: https://cs.wikipedia.org/w/index.php?title=Wikipedie%3AN%C3%A1st%C4%9Bnka%2FWikimedia_%C4%8CR%2FEditaton_v_%C4%8Cesk%C3%BDch_Bud%C4%9Bjovic%C3%ADch_%28podzim_2018%29&type=revision&diff=16569665&oldid=16547476
